spec/models/report_template_spec.rb

```diff
@@ -91,6 +91,10 @@
       expect { subject }.to change(described_class, :count).by(1)
     end
 
+    it 'has "is_duplicated_from_other_session" flag on' do
```

Comment:
I feel like `it 'has "is_duplicated_from_other_session" flag on'` implies that the value of this flag should be set to `true`

Reply:
to much `ctr+c` `ctr+v` 🙈